packages/flutterfire_ui/lib/src/auth/screens/email_verification_screen.dart

```diff
@@ -74,8 +75,8 @@ class _EmailVerificationScreenContent extends StatefulWidget {
 
   const _EmailVerificationScreenContent({
     Key? key,
-    this.auth,
-    this.actionCodeSettings,
+    required this.auth,
+    required this.actionCodeSettings,
```

Comment:
I added the `required` operator to ensure a value (no matter if null or non-null) is passed 👍